fullscreen.php

```diff
@@ -2,6 +2,7 @@
 <html>
   <head>
     <?php require get_template_directory() . '/inc/template-head-generic.php'; ?>
+    <script src="//cdn1.readspeaker.com/script/8419/webReader/webReader.js?pids=wr" type="text/javascript"></script>
```

Comment:
Instead of repeating the script to all pages, could this be moved into some generic template file?

Reply:
Fixed